client/src/component/Header.js

```diff
@@ -47,13 +49,30 @@ export const ADMIN_LINKS = [
     label: "Home",
     href: "/admin",
   },
+  {
+    label: "Menu",
+    href: "/admin/menu",
+  },
   {
     label: "Messages",
     href: "/admin/messages",
   },
 ];
-export default function Header({ links }) {
+export default function Header({ links, isAdminPage }) {
   const { isOpen, onToggle } = useDisclosure();
+  const { isAuthenticated, buildLogoutUrl, buildAuthorizeUrl } = useAuth0();
+  const [authUrl, setAuthUrl] = useState("");
+  useEffect(() => {
+    (async () => {
+      setAuthUrl(
+        await (isAuthenticated
+          ? buildLogoutUrl()
+          : buildAuthorizeUrl({
```

Comment:
Are these two functions async? I don't think they need to be awaited.

Reply:
buildAuthorizeUrl returns a Promise